comic_dl/sites/mangaFox.py

```diff
@@ -37,11 +37,21 @@ def name_cleaner(self, url):
     def single_chapter(self, comic_url, comic_name, download_directory, conversion, delete_files):
         source, cookies_main = globalFunctions.GlobalFunctions().page_downloader(manga_url=comic_url)
 
```

Comment:
Thanks for the PR. But, could you please author your code? like "@darodi for fixing #IssueNumber". Makes it easier to see which section was worked on by whom and what it fixes/adds/updates.

Reply:
done, the commit message has been changed.